app-backup/luckybackup/luckybackup-0.4.9.recipe

```diff
@@ -0,0 +1,93 @@
+SUMMARY="...a powerful, fast and reliable backup & sync tool"
+DESCRIPTION="an application that backs-up and/or synchronizes any directories with the power of rsync.\
+It is simple to use, fast (transfers over only changes made and not all data),\
+safe (keeps your data safe by checking all declared directories before proceeding in any data manipulation ),\
+reliable and fully customizable."
```

Comment:
Wrap the DESCRIPTION into 80 characters per line

Reply:
ok done